pyproject.toml

```diff
@@ -2,6 +2,7 @@
 
 requires = ["setuptools",
             "setuptools_scm",
-            "wheel"]
+            "wheel",
+            "voila"]
```

Comment:
👎 on this because it introduces voila as build-time dependency even though it is only needed for the standalone app part.

But you say we don't need this if some stuff is moved upstream? If that is the case, should we move the stuff upstream and not deal with this at all?